distribution/src/bin-regular/common.sh

```diff
@@ -3,7 +3,9 @@ if [ -z "$SCRIPT_DIR" ]; then
   exit 1;
 fi;
 
-export HAZELCAST_HOME="$(cd "$SCRIPT_DIR/.." && pwd)"
+if [ -z "$HAZELCAST_HOME"]; then
+  export HAZELCAST_HOME="$(cd "$SCRIPT_DIR/.." && pwd)"
+fi
```

Comment:
Who and why would define the `HAZELCAST_HOME` variable and why would it be different to the parent of the `SCRIPT_DIR`?

Reply:
I had to do it when trying to run the script directly from the `hazelcast` repo to test that it worked.

Reply:
So can we revert it now? I wouldn't like to add yet another variable the scripts take into account (we have quite a few already).